webapp/accounts/admin.py

```diff
@@ -142,6 +142,18 @@ def phone(self):
     return self.userprofile.phone_number
 
 
+def has_first_payment(self):
+    try:
+        return Payment.objects.filter(succeeded=True, order__user=self)\
```

Comment:
Ik denk dat deze simpeler kan, ongeveer dit:

```
return Payment.objects.filter(succeeded=True, order__user=self).exists()
```

Reply:
Oh man, so true. I was struggling with the `list_filter` part that I never looked twice.